backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -35,4 +45,40 @@ class CardMutationService {
         }
         return true
     }
+
+    @GraphQLDescription("Activate a dynamic EAK")
```

Comment:
EAK = Ehrenamtskarte.
We should also consider other entitlement cards (like the Nürnberg Pass) in this description.

```suggestion
    @GraphQLDescription("Activate a dynamic entitlement card")
```